src/contracts/interfaces/IDelegationManager.sol

```diff
@@ -87,10 +87,16 @@ interface IDelegationManager {
      */
     event OperatorMetadataURIUpdated(address indexed operator, string metadataURI);
 
-    // @notice Emitted when @param staker delegates to @param operator.
+    /// @notice Emitted whenever an operator's shares are increased for a given strategy
+    event OperatorSharesIncreased(address indexed operator, address staker, IStrategy strategy, uint256 shares);
+
+    /// @notice Emitted whenever an operator's shares are decreased for a given list of strategies
+    event OperatorSharesDecreased(address indexed operator, address staker, IStrategy[] strategies, uint256[] shares);
```

Comment:
why increase is by one strategy, but decrease is by a list of strategy?

Reply:
gas

Reply:
IMHO this is potentially a bad idea from data perspective, better keep APIs uniformed - either all single strategy, or all list of strategy

Reply:
we could always emit events in a loop when decrementing. I'm OK with that.